Gems/Atom/Tools/AtomToolsFramework/Code/Source/GraphView/GraphView.cpp

```diff
@@ -149,165 +152,173 @@ namespace AtomToolsFramework
+        m_actionAlignTop = makeAction("menuEdit", tr("Align Top"), [this](){
             GraphCanvas::AlignConfig alignConfig;
             alignConfig.m_horAlign = GraphCanvas::GraphUtils::HorizontalAlignment::None;
             alignConfig.m_verAlign = GraphCanvas::GraphUtils::VerticalAlignment::Top;
             alignConfig.m_alignTime = GetAlignmentTime();
             AlignSelected(alignConfig);
         }, {});
-        m_actionAlignBottom = makeAction(menuEdit, tr("Align Bottom"), [this](){
+        m_actionAlignBottom = makeAction("menuEdit", tr("Align Bottom"), [this](){
             GraphCanvas::AlignConfig alignConfig;
             alignConfig.m_horAlign = GraphCanvas::GraphUtils::HorizontalAlignment::None;
             alignConfig.m_verAlign = GraphCanvas::GraphUtils::VerticalAlignment::Bottom;
             alignConfig.m_alignTime = GetAlignmentTime();
             AlignSelected(alignConfig);
         }, {});
-        m_actionAlignLeft = makeAction(menuEdit, tr("Align Left"), [this](){
+        m_actionAlignLeft = makeAction("menuEdit", tr("Align Left"), [this](){
             GraphCanvas::AlignConfig alignConfig;
             alignConfig.m_horAlign = GraphCanvas::GraphUtils::HorizontalAlignment::Left;
             alignConfig.m_verAlign = GraphCanvas::GraphUtils::VerticalAlignment::None;
             alignConfig.m_alignTime = GetAlignmentTime();
             AlignSelected(alignConfig);
         }, {});
-        m_actionAlignRight = makeAction(menuEdit, tr("Align Right"), [this](){
+        m_actionAlignRight = makeAction("menuEdit", tr("Align Right"), [this](){
             GraphCanvas::AlignConfig alignConfig;
             alignConfig.m_horAlign = GraphCanvas::GraphUtils::HorizontalAlignment::Right;
             alignConfig.m_verAlign = GraphCanvas::GraphUtils::VerticalAlignment::None;
             alignConfig.m_alignTime = GetAlignmentTime();
             AlignSelected(alignConfig);
         }, {});
 
-        menuView->addSeparator();
-        m_actionPresetEditor = makeAction(menuView, tr("Preset Editor"), [this](){ OpenPresetsEditor(); }, {});
+        makeSeperator("menuView");
+        m_actionPresetEditor = makeAction("menuView", tr("Preset Editor"), [this](){ OpenPresetsEditor(); }, {});
 
-        menuView->addSeparator();
-        m_actionShowEntireGraph = makeAction(menuView, tr("Show Entire Graph"), [this](){
+        makeSeperator("menuView");
+        m_actionShowEntireGraph = makeAction("menuView", tr("Show Entire Graph"), [this](){
             GraphCanvas::ViewId viewId;
             GraphCanvas::SceneRequestBus::EventResult(viewId, m_activeGraphId, &GraphCanvas::SceneRequests::GetViewId);
             GraphCanvas::ViewRequestBus::Event(viewId, &GraphCanvas::ViewRequests::ShowEntireGraph);
-        }, QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::DownArrow));
-        m_actionZoomIn = makeAction(menuView, tr("Zoom In"), [this](){
+        }, QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_Down));
```

Comment:
Just double checking that this key sequence change was intended?

Reply:
Yes. That was intentional. It was a bug fix while testing different shortcuts to make sure that they only triggered when the correct window was focused and not, for example, in the asset browser or some other context. The shortcut did not appear in the main menu and was not working before.